src/typeahead/test/typeahead.spec.js

```diff
@@ -620,8 +636,6 @@ describe('typeahead tests', function () {
 
       // Note that this bug can only be found when element is in the document
       $document.find('body').append(element);
-      // Extra teardown for this spec
-      this.after(function () { element.remove(); });
```

Comment:
I put both test inside a describe to do that, I got the same behavior (tests passing).